openrtb-core/src/main/protobuf/openrtb.proto

```diff
@@ -758,6 +762,10 @@ message BidRequest {
     // For example, "foo.com".
     optional string domain = 3;
 
+    // The taxonomy in use.
+    // Refer to the enum com.iabtechlab.adcom.v1.enums.CategoryTaxonomy for values.
```

Comment:
the spec also says `If no cattax field is supplied IAB Cotent Category Taxonomy 1.0 is assumed.`, but I don't think it makes a difference.